src/Atmos/Model/moisture.jl

```diff
@@ -239,3 +240,131 @@ function flux_diffusive!(moist::EquilMoist, flux::Grad, state::Vars, d_q_tot)
     flux.ρu += d_q_tot .* state.ρu'
     flux.moisture.ρq_tot += d_q_tot * state.ρ
 end
+"""
+  NonEquilMoist
+
+  Assumes that there isn't any thermodynamic equilibrium
```

Comment:
Include comment on additional state variables in description

Reply:
Maybe it's better to say something like: Assumes relaxation to equilibrium state with constant relaxation timescale?